src/main/java/org/jenkinsci/backend/ircbot/IrcListener.java

```diff
@@ -750,15 +826,15 @@ private boolean addGitHubCommitter(Channel channel, User sender, String collabor
             GitHub github = GitHub.connect();
             GHUser c = github.getUser(collaborator);
             GHOrganization o = github.getOrganization(IrcBotConfig.GITHUB_ORGANIZATION);
-            
+
             final GHTeam t;
                 GHRepository forThisRepo = o.getRepository(justForThisRepo);
                  if (forThisRepo == null) {
                      out.message("Could not find repository:  "+justForThisRepo);
                      return false;
                  }
-                 t = getOrCreateRepoLocalTeam(o, forThisRepo);
-                
+                 t = getOrCreateRepoLocalTeam(o, forThisRepo, null);
```

Comment:
Won't passing `null` here cause an issue down in `getOrCreateRepoLocalTeam` when `githubUsers.isEmpty()` is called?

Reply:
Thanks missed, in the last change